app/controllers/list_controller.rb

```diff
@@ -0,0 +1,43 @@
+class ListController < ApplicationController
+  def index
+    @lists = List.all
+  end
+
+  def edit
+    @list = List.find_by_id(params[:id])
+  end
+
+  def new
+    @list = List.new
+  end
+
+  def show
+    @list = List.find_by_id(params[:id])
+  end
+
+  def create
+    @list = List.new(list_params.merge(therapist: current_user.role))
+    if @list.save
+      render json: {list: @list}, status: :ok
+    else
+      render json: {errors: @list.errors.full_messages}, status: :bad_request
```

Comment:
This is a JSON API's create method - my understanding is that list creation will be carried out synchronously.

Best practices for a synchronous `create` are to redirect to the new record's `show` page on success or to re-display the `edit` page (which will display errors) on failure.